zombienet/rococo-local-with-integritee-and-shell.toml

```diff
@@ -0,0 +1,54 @@
+[settings]
+timeout = 10000
+
+[relaychain]
+chain = "rococo-local"
+default_command = "~/code/bin/polkadot-v0.9.43"
```

Comment:
I could not make it work with relative paths, here we need to find a consensus for the location.

Reply:
I would prefer `~/bin/polkadot-v0.9.43`

Reply:
`~/.local/bin`? on my wsl I don't have a `~/bin`

Reply:
changed it to `~/bin`